stetho/src/test/java/com/facebook/stetho/json/ObjectMapperTest.java

```diff
@@ -69,14 +70,23 @@ public void testNestedProperty() throws JSONException {
     njp.child1.testString = "testString";
     njp.child2.i = 4;
 
-    String expected = "{\"child1\":{\"testString\":\"testString\"},\"child2\":{\"i\":4}}";
-    NestedJsonProperty parsed = mObjectMapper.convertValue(
-        new JSONObject(expected),
+    // The ordering of serialization changes depending on Java 7 vs Java 8.
+    String expected7 = "{\"child1\":{\"testString\":\"testString\"},\"child2\":{\"i\":4}}";
+    String expected8 = "{\"child2\":{\"i\":4},\"child1\":{\"testString\":\"testString\"}}";
```

Comment:
Why wouldn't we just do an order-independent comparison?  The order of fields is not important in JSON.  It would seem to me like linking in something like https://github.com/skyscreamer/JSONassert would be the right way to go.  This can be done easily by modifying stetho/build.gradle and adding testCompile 'org.skyscreamer:jsonassert:1.3.0' and then just using the code.  This also looks nice: https://github.com/hertzsprung/hamcrest-json.